modbus_tk/modbus.py

```diff
@@ -338,6 +365,13 @@ def execute(
                             digits.append(byte_val % 2)
                             byte_val = byte_val >> 1
                     result = tuple(digits)
+                if function_code == defines.READ_FILE_RECORD:
+                    sub_seq=list()
```

Comment:
The code should follow the coding style of the project and have one space between operators

For examplle : `ptr < len(results)`, `sub_seq += ((ptr + 2, ptr + 2 + results[ptr] // 2`, `ptr+=result[ptr] // 2 + 2`, `result=tuple(map(lambda x: result[x[0]:x[1]], sub_seq))`

varibles should be at lealt 3 letters (x in lambda is not correct). The name should be "explicative"

Reply:
Accepted, resolved in the next commits